src/components/common/ConfirmAction.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ButtonComponent from './ButtonComponent';
+
+class ConfirmAction extends React.Component {
+  shouldComponentUpdate(nextProps) {
+    const { buttonLoading } = nextProps;
+    if (!buttonLoading) nextProps.toggleModal();
+    return true;
+  }
+
+  render() {
+    return (
+      <div>
```

Comment:
You can use `Fragment` for encapsulation instead of empty `div`. I.e.
```
import React, { Fragment } from 'react';
...
...

render() {
  return (
    <Fragment>
      <label />
      ....
    </Fragment>
  );
}